res/css/views/rooms/_EventTile.scss

```diff
@@ -485,6 +490,11 @@ $left-gutter: 64px;
         color: inherit; // inherit the colour from the dark or light theme by default (but not for code blocks)
         font-size: $font-14px;
 
+        ol,
+        ul {
```

Comment:
we should add comments for what this is targeting - it's not immediately obvious what the `ol` and `ul` actually are.